images/clonerefs/Dockerfile

```diff
@@ -26,3 +26,6 @@ ENV IMAGE=${IMAGE_ARG}
 
 RUN adduser -S -h /home/prow -s /bin/bash -g root -u 1000 prow
 USER prow
+
+# Ensure .ssh folder exists so known_hosts file can be written
+RUN mkdir -p /home/prow/.ssh
```

Comment:
I don't know whether clonerefs cares, but normally you'd also then need to
```
chown -R prow:root /home/prow
```
and
```
chmod 0700 /home/prow/.ssh
```